Sources/web3swift/Web3/Web3+Instance.swift

```diff
@@ -20,29 +20,26 @@ public class Web3 {
     /// Keystore manager can be bound to Web3 instance. If some manager is bound all further account related functions, such
     /// as account listing, transaction signing, etc. are done locally using private keys and accounts found in a manager.
     public func addKeystoreManager(_ manager: KeystoreManager?) {
-        self.provider.attachedKeystoreManager = manager
+        provider.attachedKeystoreManager = manager
     }
 
-    var ethInstance: Web3.Eth?
+    var ethInstance: IEth?
 
     /// Public web3.eth.* namespace.
-    public var eth: Web3.Eth {
-        if self.ethInstance != nil {
-            return self.ethInstance!
+    public var eth: IEth {
```

Comment:
Correct me if I wrong, is this `IEth` type are replacement of `Web.eth` **subtype** one?